documentation/usage/installation.md

```diff
@@ -2,18 +2,153 @@
+
+This installer however is not signed, which may lead to warnings from Microsoft Defender or your antivirus of choice, as installing unsigned applications can be a security risk. This is unfortunately something we can not fix, as signing an application is both expensive and time consuming, and this project is being developed free of charge by a community of volunteers.
+
+If you trust this application even without the signature, some screenshots of how to pass through the most common warnings can be found below. If you have some technical experience, and would prefer to build this application from the source yourself to avoid these errors, then take a look at the [project README instructions](https://github.com/open-goal/jak-project#setting-up-a-development-environment) instead. Support for this however is outside the scope of this document.
+
+<details>
+  <summary>Expand this section to see examples warnings and ignore them (if you trust this application!)</summary>
+  <div>
+
+![The first warning](/docs/installation/executable_warning.png)
+![Windows Defender SmartScreen hides the skip button](/docs/installation/windows_defender_warning.png)
+![Run anyway if you trust this application](/docs/installation/windows_defender_run_anyway.png)
+
+  </div>
+</details>
+
+If you have chosen to continue installing the application after these warnings, you'll be shown a simple installation wizard that will allow you to configure and launch OpenGOAL. Some step by step screenshots have been included below.
+
+<details>
+  <summary>Expand this section to see the process of installing OpenGOAL</summary>
+  <div>
+
+![Launching the installer](/docs/installation/opengoal_install_start.png)
+![Choose a location to install OpenGOAL](/docs/installation/opengoal_install_location.png)
+![Start the install](/docs/installation/opengoal_install_confirm.png)
+![Wait for the installer to complete](/docs/installation/opengoal_install_in_progress.png)
+![Launch OpenGOAL once the install is complete](/docs/installation/opengoal_install_finished.png)
+
+  </div>
+</details>
+
+If you choose to launch OpenGOAL once the install is completed, you'll skip right to the next step. If not you'll need to navigate to the location you chose for OpenGOAL to install (by default this location is `C:\Program Files\OpenGOAL-Launcher`), and run `OpenGOAL-Launcher.exe`.
+
+If this opens successfully, you've reached the final stage! Up next we'll be setting up the launcher, including extracting the assets from your `.iso` file.
+
+### Setting up the launcher
+
+This step is very straightforward, but as this project is still in active development you may come across issues along the way. If you hit issues please remember that this is a community led project, and we can't guarantee support. Please first visit the [FAQs](/faq.md) and if you can't find an answer there, you might be able to find help in the OpenGOAL [discord](https://discord.gg/VZbXMHXzWv), but we do not guarantee support.
```

Comment:
maybe this would be better put near the top in an admonition, since they may run into problems on early steps as well?

Reply:
There is a copy of the warning with links to the FAQs and Discord up in the second paragraph of the `Opening notes` section, but I've added it into a highlight block and made it the first paragraph to make it more obvious. I've made that change for the commit coming next and left this paragraph as is, but if you'd like me to remove it here or make any other changes with the top paragraph once I've pushed it let me know

Reply:
I'll leave that up to you, repeating this kind of info isn't a bad thing imo cause it's often missed.